internal/service/autoscaling/policy_test.go

```diff
@@ -143,6 +143,60 @@ func TestAccAutoScalingPolicy_predictiveScaling(t *testing.T) {
 	})
 }
 
+func TestAccAutoScalingPolicy_predictiveScalingCustom(t *testing.T) {
+	var policy autoscaling.ScalingPolicy
+
+	resourceSimpleName := "aws_autoscaling_policy.test"
```

Comment:
Can we remove the extra spacing between lines?

Reply:
I checked other tests under `autoscaling`, it seemed to be a common pattern to leave a blank line above `resource.ParallelTest`, I will follow that pattern and remove other blank lines.